stlab/config.hpp.in

```diff
@@ -7,6 +7,9 @@
 #ifndef STLAB_CONFIG_HPP
 #define STLAB_CONFIG_HPP
 
+// Release version macro - used for inline namespace increment on each release
```

Comment:
Do you want to add a comment here about _not_ bumping `pre_exit`? I saw it in the PR description and wonder if it should last longer than this PR...

Reply:
The comment is in `await.hpp`.